### PR TITLE
Update local_csp & bill utils

### PR DESCRIPTION
**Pull request by smarlowucf**
with changes to meter_billing. timestamp is datetime type and added dry_run bool option.

### CODE REVIEW

**Review by rtamalin — APPROVED** [review]
LGTM

File: tests/unit/test_local_csp.py [hunk 1]
```diff
@@ -32,7 +34,7 @@
 
 def test_meter_billing_ok(cba_config):
```
Comment (KeithMnemonic) [inline tests/unit/test_local_csp.py:35]: docstrings?

File: tests/unit/test_local_csp.py [hunk 2]
```diff
@@ -47,15 +49,16 @@ def test_meter_billing_ok(cba_config):
...
                 test_dimensions,
-                test_timestamp
+                test_timestamp,
+                dry_run=False
             )
 
             assert billing_id == str(test_uuid.hex)
 
 
 def test_meter_billing_error(cba_config):
```
Comment (KeithMnemonic) [inline tests/unit/test_local_csp.py:59]: docstrings?

File: tests/unit/test_local_csp.py (hunk 1 above)
Comment (smarlowucf) [reply tests/unit/test_local_csp.py:35]: Seems unnecessary in tests?

Comment (KeithMnemonic) [reply tests/unit/test_local_csp.py:35]: every other test has a docstring now I thought. If so let's be consistent

Comment (smarlowucf) [reply tests/unit/test_local_csp.py:35]: I don't think so. I skimmed a couple other test modules and there's no function docstrings. Each one has a module docstring and there's some one line comments here and there. But not consistent docstrings. Will move forward on this for now. I don't see the need for docstrings in tests but if we want them they are needed everywhere it seems. So we could do that all at once?

Comment (smarlowucf) [reply tests/unit/test_local_csp.py:35]: Some modules do. But I found an example:

```
def test_good_get_next_bill_time_monthly():
    """Test working utils.get_next_bill_time"""
    assert utils.get_next_bill_time(
...
```

The docstring seems unnecessary. It doesn't really add any value.

Comment (KeithMnemonic) [reply tests/unit/test_local_csp.py:35]: fine but they are in test_adapter.py, test_config.py, and test_utlils.py so we should be consistent either way

**Review by KeithMnemonic — APPROVED** [review]
LGTM

File: tests/unit/test_local_csp.py (hunk 1 above)
Comment (smarlowucf) [reply tests/unit/test_local_csp.py:35]: Either with or without is fine by me.

**Pull request merged**